packages/events/src/EventBoundary.ts

```diff
@@ -136,17 +134,23 @@ export class EventBoundary
      */
     protected eventPool: Map<typeof FederatedEvent, FederatedEvent[]> = new Map();
 
+    private _eventSystem: EventSystem;
+
     /** Every interactive element gathered from the scene. Only used in `pointermove` */
     private _allInteractiveElements: FederatedEventTarget[] = [];
     /** Every element that passed the hit test. Only used in `pointermove` */
     private _hitElements: FederatedEventTarget[] = [];
     /** Whether or not to collect all the interactive elements from the scene. Enabled in `pointermove` */
     private _collectInteractiveElements = false;
 
-    /** @param rootTarget - The holder of the event boundary. */
-    constructor(rootTarget?: DisplayObject)
+    /**
+     * @param rootTarget - The holder of the event boundary.
+     * @param eventSystem - The event system that manages this boundary.
+     */
+    constructor(rootTarget?: DisplayObject, eventSystem?: EventSystem)
```

Comment:
Preference here to not passing in the EventSystem. EventBoundary only needs to know about `globalMove`.

Reply:
I think this is required as the user could override `features` entirely
```
renderer.events.features = {
    move: true,
    globalMove: false,
    click: false,
    wheel: false
}
```
So in order to not lose the reference I need to pass the EventSystem?

Reply:
The goal here is to limit the scope and prevent coupling of EventSystem and EventBoundary. EventBoundary not knowing anything about EventSystem is a much cleaner design.

There's no reason we should allow re-assignment of `features`, only set individual props. I don't think there's anything wrong with that. Devs can use Object.assign, if need be. I think the most common case is turning off a flag.

Second, you could make features a Proxy (like Mat just did in Assets), and handle setting `globalMove` to update a property on EventBoundary:

```js
this.features = new Proxy({...EventSystem.defaultEventFeatures}, {
  set: (target, key, value) => {
    if (key === 'globalMove') {
       this.rootBoundary.globalMoveFeature = value;
    }
    target[key] = value;
    return true;
  }
);
```

Reply:
Yep forgot we could use proxies now!